scripts/bump-version.sh

```diff
@@ -7,7 +7,7 @@ NEW_VERSION="$2"
 WAL2JSON_VERSION=${WAL2JSON_VERSION:-$(curl -s "https://api.github.com/repos/getsentry/wal2json/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')}
 
 sed -i -e "s/^WAL2JSON_VERSION=\([^:]\+\):.\+\$/WAL2JSON_VERSION=\1:$WAL2JSON_VERSION/" .env
-sed -i -e "s/^\(SENTRY\|SNUBA\|RELAY\|SYMBOLICATOR\)_IMAGE=\([^:]\+\):.\+\$/\1_IMAGE=\2:$NEW_VERSION/" .env
+sed -i -e "s/^\(SENTRY\|SNUBA\|RELAY\|SYMBOLICATOR|VROOM\)_IMAGE=\([^:]\+\):.\+\$/\1_IMAGE=\2:$NEW_VERSION/" .env
```

Comment:
Maybe we should consider making this a blocklist now since almost all images get updated?

Reply:
https://github.com/getsentry/self-hosted/pull/2207 ftr, missing escape of the `|` led to [a corrupt 23.6.0 release](https://github.com/getsentry/self-hosted/blob/23.6.0/.env#L11-L12). 🐭